tools/behave_rule.bzl

```diff
@@ -64,7 +64,7 @@ def _rule_implementation(ctx):
            echo Starting Grakn Server
            mkdir ./grakn_distribution/"$DIRECTORY"/grakn_test
            ./grakn_distribution/"$DIRECTORY"/grakn server --data grakn_test &
-           sleep 8
+           sleep 9
```

Comment:
This is a simple CI fix but in a follow-up PR, I will be changing this logic to poll `lsof` until the server has actually started.